community/samples/serving/helloworld-java-quarkus/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# https://github.com/redhat-developer-demos/quarkus-java-builder allows to set 
+# various maven build arguments such as MAVEN_MIRROR_URL, HTTP_PROXY_HOST etc.,
+# it also allows building images with https://buildah.io
+
+FROM quay.io/rhdevelopers/quarkus-java-builder:graal-1.0.0-rc14 as nativebuilder
+COPY . /project
+
+# uncomment this to set the MAVEN_MIRROR_URL of your choice, to make faster builds
+# ARG MAVEN_MIRROR_URL=<your-maven-mirror-url>
+# e.g. 
+ARG MAVEN_MIRROR_URL=http://192.168.0.107:8081/nexus/content/groups/public
+
+RUN  /usr/local/bin/entrypoint-run.sh mvn -DskipTests clean package -Pnative 
+
+FROM registry.fedoraproject.org/fedora-minimal
```

Comment:
Nice to see some more container registries. For minimal points of failure, is fedora-minimal available on quay.io? Or perhaps this could be moved to use Debian or Alpine from Docker Hub?

Reply:
I don't think that's in our control, since this is a community contribution, and rh have already use quay.io to host their image.  I don't think we'd move this else where.